src/Examples/Examples/ResponseExtensions.cs

```diff
@@ -17,6 +17,13 @@ public static class ResponseExtensions
 	{
 		private static readonly JsonSerializer Serializer = new JsonSerializer();
 
+		public static void MatchesExample(this IResponse response, string content, Action<Example, JObject> clientChanges) =>
```

Comment:
Really not keen on this! Seems overly convoluted to save typing an extra line

Reply:
It's not just about saving one line

https://github.com/elastic/elasticsearch-net/pull/4239/commits/19a0c1c1ea3ee86d7b39c34185d1c8ebbca41bfd#diff-8c3e51d6895a9329f7e6602bb0c00754L69-L81

Saves on explicit return (we never return a new Example), and additional indentation, nested lambda's.